src/translations/sqlb_de.ts

```diff
@@ -3101,12 +3101,12 @@ Sie können SQL-Statements aus einer Objektzeile fassen und in anderen Anwendung
     <message>
         <location filename="../MainWindow.ui" line="1966"/>
         <source>Save as &amp;view</source>
-        <translation>Als &amp;View speichern</translation>
+        <translation>Als &amp;Ansicht speichern</translation>
```

Comment:
This replaces the keyboard shortcut. Have you verified that there ideally are no duplicates?